index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./CSS/style.css">
+</head>
+<body>
+  <div class="container">
+    <!-- Header  -->
+    <header>
+       <nav class="top-nav">
+            <h1 class="logo">My log</h1>
+          <ul>
```

Comment:
-[OPTIONAL]  Kindly make sure the logo is in a <a> tag as per the general convention and best practices :recycle: